sda/service.go

```diff
@@ -434,11 +434,14 @@ func (c *Client) Send(dst *network.ServerIdentity, msg network.Body) (*network.P
 		// Catch an eventual error
 		err := ErrMsg(&response, nil)
 		if err != nil {
+			log.Lvl4("Closing connection to", dst)
 			return nil, err
 		}
+		log.Lvl4("Closing connection to", dst)
 		return &response, nil
-	case <-time.After(time.Second * 10):
+	case <-time.After(time.Minute * 30):
```

Comment:
wow wow wow. You also changed a value in a separate PR in client. let's put someting reasonable please :)